src/routes/index.js

```diff
@@ -1,14 +1,14 @@
 import express from 'express';
 import authRoute from './auth.route';
 import tripRoute from './trip.route';
-import profileSettingsRoute from './profile-settings.route';
 import accommodationRoute from './accommodation.route';
+import SettingsRoute from './settings.route';
 
 const app = express();
 
 app.use('/api/auth', authRoute);
 app.use('/api/', tripRoute);
-app.use('/api/', profileSettingsRoute);
 app.use('/api/accommodations', accommodationRoute);
+app.use('/api/users', SettingsRoute);
```

Comment:
Use `/api/users/settings` as the route context here

Reply:
@adesege Ok solved